static/css/v3/library-highlight-carousel.css

```
@@ -0,0 +1,213 @@
+  min-width: 0;
+}
+
+/* Hidden radios are the single source of carousel state. */
+.library-highlight-carousel__radio {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)